spring-boot-tools/spring-boot-loader-tools/src/main/java/org/springframework/boot/loader/tools/RunProcess.java

```diff
@@ -58,6 +66,7 @@ public int run(boolean waitForProcess, String... args) throws IOException {
 	protected int run(boolean waitForProcess, Collection<String> args)
 			throws IOException {
 		ProcessBuilder builder = new ProcessBuilder(this.command);
+		builder.directory(workingDirectory);
```

Comment:
I've updated the PR (+rebase). Also I've tried to add some javadoc comments. Writing comments (especially in English) is not my forte, so fell free to edit them :smile: